xdsl/ir.py

```diff
@@ -211,6 +212,8 @@ class Attribute(ABC):
     Attributes are used to represent SSA variable types, and can be attached
     on operations to give extra information.
     """
+    is_Parametrized = False
```

Comment:
Do we really need this?
Is this supposed to remove the `isinstance` checks? I would prefer keeping the `isinstance` personally.
For instance, this allows the type checker to understand what you are doing.

Reply:
ok, I will revert these changes